docs/sdks/client-side-identity.md

```diff
@@ -248,11 +260,11 @@ Initializes the SDK and establishes user identity for targeted advertising.
 
 Here's what you need to know about this function:
 
-- You can call `init()` any time after the SDK has been loaded. The recommended way to do this is by registering a callback function which handles the `SdkLoaded` event using the [Array Push Pattern](#array-push-pattern). Using this pattern means your code will work regardless of script load order, and using `async` or `defer` on your script tags will not cause UID2 SDK errors.
+- You can call `init()` any time after the SDK has been loaded. The recommended way to do this is by registering a callback function that handles the `SdkLoaded` event using the [Array Push Pattern](#array-push-pattern). By using this pattern you can make sure that your code works regardless of script load order, and that using `async` or `defer` on your script tags does not cause UID2 SDK errors.
 - The `identity` property in the `init()` call refers to the `body` property of the response JSON object returned from a successful [POST /token/generate](../endpoints/post-token-generate.md) or [POST /token/refresh](../endpoints/post-token-refresh.md) call with the server-side generated identity. This is a good way to provide the identity if your server-side integration ensures you always have a current token available and it is more convenient to provide it using JavaScript.
-- If the `identity` property in the `init()` call is falsy, the SDK will attempt to load the identity from local storage or the cookie.
-  - Once `init()` is complete, all callbacks will receive the `InitCompleted` event. If the `identity` property on the payload of this event is null, no identity could be loaded and you should [provide a valid identity](#provide-identity). This is the recommended way to provide an identity if your server-side integration does not ensure a current identity is always available, and you need to request it from the server only when necessary.
-  - If you are using a [first-party cookie](#uid2-cookie-format) to store the passed UID2 information for the session, a call to `init()` made by a page on a different domain might not be able to access the cookie. You can adjust the settings used for the cookie with the `cookieDomain` and `cookiePath` options.
+- If the `identity` property in the `init()` call is false, the SDK attempts to load the identity from local storage or the cookie.
```

Comment:
This must be `falsy` and not `false` - it's a specific JavaScript/TypeScript language term which means something specific.

Reply:
See https://developer.mozilla.org/en-US/docs/Glossary/Falsy for details if you're interested 🙂